### PR TITLE
Last of the Open Source Changes

### PR DESCRIPTION
**Pull request by DrGirlfriend**
Added to readme
Changed out all internal links

Next step to publish to NPM
Tests that fail are due to timeouts (mocha is set to 20000ms currently)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,69 @@ const metrics = new NewRelic({
...
+| `durationConnect` | number | TCP connection duration in milliseconds | `1.138568` |
+| `durationSSL` | number | SSL handshake duration in milliseconds (https only) | `3.639361` |
+| `durationSend` | number | time it took to send the HTTP request. Note this currently does not work for streaming requests as done for our rendition uploads until we upgrade to Node 12+ in Adobe I/O Runtime. Until then the send time is included in the `durationConnect`. | `0.065885` |
+| `durationWait` | number | time between request was sent and first byte of response was received in milliseconds | `288.649511` |
+| `durationReceive` | number | time it took to receive the entire response body in milliseconds | `0.029158` |
+| `error` | string | Only set if there was a low-level connection error. Set as `true` in json, represented as `1` in NewRelic. | `1` |
+| `errorCode` | string | OS or nodejs error code (name or number) in case there was a low-level connection error. `110` means `ETIMEDOUT`. | `"ECONNRESET"` or `"110"` |
+| `errorMessage` | string | Error message in case there was a low-level connection error. | `"socket hang up"` |
+
+### Standard
```
Comment (alexkli) [inline README.md:169]: Maybe have the "Standard" section come before the "Http" one.

File: README.md [hunk 2]
```diff
@@ -128,6 +128,69 @@ const metrics = new NewRelic({
...
+| `requestId`       | string           | x-request-id of the original incoming http request |
+| `namespace`       | string           | OpenWhisk namespace of the action that sent the event. |
+| `package`         | string           | OpenWhisk package name of the action that sent the event.  |
+| `actionName`      | string           | OpenWhisk action name (without package) of the action that sent the event. |
+| `activationId`    | string           | OpenWhisk activation id of the action that sent the event. |
+| `orgId`           | string           | IMS organization id of the invoking technical account/integration/client. |
+| `appName`         | string           | Name of the invoking IO console integration (API key label) |
+| `clientId`        | string           | API key = IMS client id of the invoking technical account/integration/client. |
+| `sourceName`      | string           | Filename of the source. |
+| `sourceSize`      | number           | Size in bytes of the source. |
```
Comment (alexkli) [inline README.md:186]: `orgId` to `sourceSize` can be removed as they are not set by this library, they are custom Nui specific and set in `asset-compute-commons`.

File: README.md [hunk 3]
```diff
@@ -128,6 +128,69 @@ const metrics = new NewRelic({
...
+
+### Standard
+
+Sent for all metrics.
+
+| Attribute         | Format           | Description                     |
+|-------------------|------------------|---------------------------------|
+| `eventType`       | string           | Event type, required, [standard New Relic Insights type](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/insights-custom-data-requirements-limits). |
+| `timestamp`       | utc millis (?)   | The UTC timestamp to associate with the event. [Standard New Relic Insights type](https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/insights-custom-data-requirements-limits). |
+| `requestId`       | string           | x-request-id of the original incoming http request |
```
Comment (alexkli) [inline README.md:177]: `requestId` can be removed as it's not set by this library.

File: package.json [hunk 4]
```diff
@@ -38,7 +38,7 @@
     "adobe"
   ],
   "scripts": {
-    "test": "nyc --reporter=html --reporter=text mocha test --timeout 60000 --recursive && npm run checkout-node-fetch && npm run test-node-fetch",
+    "test": "nyc --reporter=html --reporter=text mocha test --timeout 20000 --recursive && npm run checkout-node-fetch && npm run test-node-fetch",
```
Comment (jdelbick) [inline package.json:41]: why are you decreasing the timeout if tests were failing due to timeouts?

**Comment by jdelbick** [comment]
qs are why are we releasing to npm this as @adobe/node-openwhisk-newrelic instead of @adobe/openwhisk-newrelic https://github.com/adobe/node-openwhisk-newrelic/blob/master/package.json#L2 ?

File: package.json (hunk 4 above)
Comment (DrGirlfriend) [reply package.json:41]: @jdelbick I tried both, in the retro the notes say 20000. Either value above and including 600000 failed as timeouts

Comment (tmathern) [reply package.json:41]: No need to reduce the value if it timeouts :)
20000ms is a start value, you can increase over that. 
More interesting would be to understand why it takes so long.

**Comment by DrGirlfriend** [comment]
No clue and they are all random; e.g. sometimes Mac Nodejs 14 will fail a test complaining about ticks, and then sometime Mac Nodejs 10.15 will fail in the same manor. Also there could be 1-3 tests that endup timing out

I did read several people have this same issue even when Travis systems are all green. They mentioned it is related to network traffic and containers.

But what seems really strange is that Linux did not exhibit this for the run's I looked at. So maybe it's a mac container thing?

**Comment by tmathern** [comment]
Often, it depends on how the operating system handles file and network accesses (and this can change in-between operating systems).

**Comment by DrGirlfriend** [comment]
@tmathern Re ran 3 failed tests and all came back green

**Comment by DrGirlfriend** [comment]
Published https://www.npmjs.com/package/@adobe/openwhisk-newrelic

**Pull request merged**